packages/access-api/test/provider-add.test.js

```diff
@@ -97,6 +99,100 @@ for (const accessApiVariant of /** @type {const} */ ([
       })
     })
   })
+
+  if (
+    ['provider/add', 'access/delegate'].every((f) =>
+      accessApiVariant.features.has(f)
+    )
+  ) {
+    it('provider/add allows for access/delegate', async () => {
+      const space = await principal.ed25519.generate()
+      const issuer = await accessApiVariant.issuer
+      const service = await accessApiVariant.audience
+      const accountDid = /** @type {const} */ ('did:mailto:example.com:foo')
+      const serviceSessionAttest = await ucanto.delegate({
+        issuer: service,
+        audience: issuer,
+        capabilities: [
+          {
+            can: 'provider/add',
+            with: accountDid,
+          },
+          {
+            can: 'access/delegate',
+            with: space.did(),
+          },
+        ],
+      })
+      const sessionProofs = [
```

Comment:
Not sure I understand what you're trying to do here, but `ucan/attest` should only be considered when validating ucan issued by non did:key and an issuer different from the service.

UCAN been attested does not meet this criteria so it should have no effect on the validator.

Reply:
I'm trying to test whatever we expect to use for this https://github.com/web3-storage/w3cli/issues/46

Reply:
https://github.com/web3-storage/w3protocol/pull/492/files#diff-b90906f041e93cd2ae9feab2b5b6e0e6b2cf0ac36d5917b5e2e67cdbf04b406cR108